test/multiverse/suites/elasticsearch/elasticsearch_instrumentation_test.rb

```diff
@@ -0,0 +1,176 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require 'elasticsearch'
+require 'socket'
+
+class ElasticsearchInstrumentationTest < Minitest::Test
+  def setup
+    # Keeping the log off prevents noisy test output
+    @client = ::Elasticsearch::Client.new(
+      log: false,
+      hosts: "localhost:#{port}"
+    )
+    # Ensure the client is running before the tests start
+    @client.cluster.health
+    # TODO: Update this to use constants, perhaps a hash?
```

Comment:
I think we can probably remove this TODO since the index arguments are only passed once